scalajvm/public/stylesheets/main.css

```diff
@@ -94,24 +96,16 @@ ul.menu {
 	background: #445566;
 }
 
-.quote-time {
-    float: right;
-}
```

Comment:
You have removed this class, but it's still used in `index.scala.html` and `quote.scala.html`. Please review this action.